lesson6.js

```diff
@@ -1,7 +1,13 @@
 const ul = document.getElementById('js-list');
+const loadingTime = document.createElement('img');
+loadingTime.src = "loading-circle.gif";
+loadingTime.id = 'loading-img'
+ul.appendChild(loadingTime);
```

Comment:
IMO
I think this part can be functionalized.

Reply:
I'm going to create a function called makeLoadingImage to make it work!